MobileCenter/MobileCenter/Internals/Application Delegate/MSAppDelegateForwarder.m

```diff
@@ -286,19 +287,28 @@ - (void)custom_application:(UIApplication *)app
     didReceiveRemoteNotification:(NSDictionary *)userInfo
           fetchCompletionHandler:(void (^)(UIBackgroundFetchResult))completionHandler {
 
+  // Collect `UIBackgroundFetchResult` from delegates in order to call the original completion handler later.
+  __block UIBackgroundFetchResult forwardedFetchResult = UIBackgroundFetchResultNoData;
+  void (^forwardedCompletionHandler)(UIBackgroundFetchResult) = ^(UIBackgroundFetchResult fetchResult) {
+    forwardedFetchResult = fetchResult;
```

Comment:
Looks it overwrites whatever the previous fetch results are. In other words, if there are multiple methods and it returned different value, it will be replaced by the last return value. Is this possible?